install_config/configuring_local.adoc

```diff
@@ -253,23 +269,23 @@ kubeletArguments:
    - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true,BlockVolume=true
 ----
 
-. Restart the master. The nodes should be restarted automatically after the
-configuration change (it may take several minutes).
+. Restart the master. The nodes should restart automatically after the
+configuration change (this may take several minutes).
 
 [[local-volume-prepare-block-devices]]
-==== Prepare the Block Devices
-Before starting the provisioner all the block devices that should be available
-to the pods need to be linked to the *_/mnt/local-storage/<storage class>_* directory
-structure. Example: to make a device *_/dev/dm-36_* available:
+==== Preparing the Raw Block Devices
+Before starting the provisioner, all raw block devices that should be available
```

Comment:
Maybe
Before you start the provisioner, link all the raw block devices that pods can use to the *_/mnt/local-storage/<storage class>_* directory structure.